feat(docs): Release adoption stages filtering (#3925)
getsentry/sentry-docs: src/docs/product/releases/health/index.mdx

The chart does not display if you don't select a project or if you select multiple projects.

### Adoption Stages

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

Adoption stage is a high-level overview of the usage your release is seeing relative to other releases. It is calculated every hour by processing your sessions over the last six hours.

> For each release, the release's percentage of sessions against all sessions in the same environment and project is calculated and marked as being in one of three stages based on a threshold of 10%. These stages are:

Reviewer: ```suggestion
For each release, the release's percentage of sessions is calculated against all sessions in the same environment and project. It's then marked as being in one of three stages, based on a threshold of 10%. These stages are:
```

> - "Adopted" - the release's percentage of sessions across all sessions is above the threshold, ie. the release was seen in 10% or more of sessions over all the releases in the last six hours.

Reviewer: ```suggestion
- Adopted - The release's percentage of sessions compared to all sessions is above the threshold. That is, the release was seen in 10% or more of sessions over all releases in the last six hours.
```

> - "Low Adoption" - the release's percentage of sessions across all session that are below the threshold, ie. the release was seen in less than 10% of sessions over all the releases in the last six hours.

Reviewer: ```suggestion
- Low Adoption - The release's percentage of sessions compared to all sessions that are below the threshold. That is, the release was seen in less than 10% of sessions over all the releases in the last six hours.
```

> - "Replaced" -  the release was previously tagged as "Adopted", but no longer meets the minimum threshold. "Replaced" releases that reach the threshold will once again be marked as "Adopted".

Reviewer: ```suggestion
- Replaced -  The release was previously tagged as "Adopted", but no longer meets the minimum threshold. "Replaced" releases that reach the threshold will once again be marked as "Adopted".
```

<Note>

> Adoption Stage labels are only displayed when filtered down to a single environment.

Reviewer: ```suggestion
Adoption Stage labels are only displayed when you filter down to a single environment.
```
